Detectors/ITSMFT/MFT/assessment/include/MFTAssessment/MFTAssessment.h

```diff
@@ -32,6 +32,12 @@
 #include "Steer/MCKinematicsReader.h"
 #include <unordered_map>
 #include <vector>
+#include "DataFormatsITSMFT/TopologyDictionary.h"
+#include "MFTTracking/IOUtils.h"
+#include "ReconstructionDataFormats/BaseCluster.h"
+#include "ITSMFTReconstruction/ClustererParam.h"
+#include "DetectorsCommonDataFormats/DetectorNameConf.h"
+#include "MFTBase/GeometryTGeo.h"
```

Comment:
Hi Sarah, thanks this PR.
Since `"MFTBase/GeometryTGeo.h"` is not needed, please remove all includes and references.